Code/Editor/EditorFramework/Assets/Implementation/AssetBrowserView.cpp

```diff
@@ -236,6 +235,9 @@ void ezQtIconViewDelegate::paint(QPainter* painter, const QStyleOptionViewItem&
       case ezAssetInfo::TransformState::TransformError:
         ezQtUiServices::GetSingleton()->GetCachedIconResource(":/EditorFramework/Icons/AssetFailedTransform16.png").paint(painter, thumbnailRect);
         break;
+      case ezAssetInfo::TransformState::NeedsImport:
+        ezQtUiServices::GetSingleton()->GetCachedIconResource(":/EditorFramework/Icons/AssetFailedTransform16.png").paint(painter, thumbnailRect); // TODO

```

Comment:
todo ?

Reply:
Added a new image for this case (just needs transform in yellow).